lib/msal-core/src/error/AuthError.ts

```diff
@@ -36,12 +36,11 @@ export class AuthError extends Error {
     errorCode: string;
     errorMessage: string;
 
-    constructor(errorCode: string, errorMessage: string) {
+    constructor(errorCode: string, errorMessage?: string) {
```

Comment:
just doublechecking that we don't wanna set a default here? does anything assume error message is required in this class / subclass